src/models/applicationState.ts

```diff
@@ -198,6 +199,23 @@ export interface IProjectVideoSettings {
     frameExtractionRate: number;
 }
 
+/**
+ * @name - Video Tagging Settings for the project
```

Comment:
Docs are incorrect.  Also - If you are going to add `@member` declaration please also add docs for each member.